src/app/packages/generic-type-resolver/generic-type-resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Injector } from '@angular/core';
+
+export type injectionResolverFilter = (resolvedClasses: any[]) => any;
+
+export class GenericTypeResolver {
+    static resolveMultiInjection(injector: Injector, token: any, filterFunction: injectionResolverFilter, isSingelton: boolean = false): any {
```

Comment:
filterFunction should be with default value of null.
It is not required parameter